weave-js/src/components/PagePanelComponents/Home/Browse3/filters/CellFilterWrapper.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Wrap a cell with an Option+Click handler that adds a filter
+ * with the corresponding field, operation, and value.
+ */
+
+import React from 'react';
+
+type CellFilterWrapperProps = {
+  children: React.ReactNode;
+  onAddFilter?: (key: string, operation: string | null, value: any) => void;
+  field: string;
+  operation: string | null;
+  value: any;
+};
+
+export const CellFilterWrapper = ({
+  children,
+  onAddFilter,
+  field,
+  operation,
+  value,
+}: CellFilterWrapperProps) => {
+  const onClickCapture = onAddFilter
+    ? (e: React.MouseEvent) => {
+        // event.altKey - pressed Option key on Macs
+        if (e.altKey) {
```

Comment:
I could see a nice little "did you know? ... " footer on the filter modal telling users the can option-click

Reply:
I think this is a good idea, let me follow up with design. Might hold off until it works for status/costs/tokens too.